src/pages/case-studies/neo-financial-improves-the-developer-experience-with-sourcegraph.tsx

```diff
@@ -0,0 +1,294 @@
+            image: 'https://storage.googleapis.com/sourcegraph-assets/about.sourcegraph.com/meta/case-study-neo.png',
+        }}
+        hero={
+            <Hero
+                backButton={{
+                    text: 'Case Studies',
+                    link: '/case-studies',
+                }}
+                variant="venusCode"
+                title={'Neo Financial improves the\n developer experience with \nSourcegraph'}
+                displayUnderNav={true}
+            />
+        }
+    >
+        <NewCaseStudyLayout customer="Neo Financial">
+            <ContentSection background="white" slimWidth={true}>
+                <Blockquote
+                    inline={true}
+                    quote="With Sourcegraph, developers are more productive and it’s clear that every team is getting 1% better every day."
+                    author="Ronnie Magatti, Team Lead & Principal Software Engineer at Neo Financial"
+                    logo={{
+                        src: '/external-logos/neo-financial.svg',
+                        alt: 'Neo Financial',
+                        href: 'https://neofinancial.com',
+                    }}
+                    border={false}
+                    largeText={true}
+                />
+            </ContentSection>
+
+            <ContentSection>
+                <UseChallengeSolutionResults
+                    useCases={[
+                        {
+                            text: 'Code Reuse',
+                        },
+                    ]}
+                    challenges={[
+                        {
+                            text: 'Unable to understand their codebase and efficiently find code to reuse with code host’s native search functionality.',
+                        },
+                        {
+                            text: 'Inability to search branches or tags at scale required manual effort and wasted time.',
+                        },
+                        {
+                            text: "Local, unstructured searches through repositories were time consuming and weren't always successful.",
+                        },
+                    ]}
+                    solutions={[
+                        {
+                            text: 'Search across commits quickly and make changes with confidence that nothing is missed',
+                        },
+                        {
+                            text: 'Track and manage codebase maintenance with Code Insights and share visual dashboards with relevant teams and stakeholders',
+                        },
+                        {
+                            text: 'Solve challenges faster with an easy and intuitive interface for code search',
+                        },
+                    ]}
```

Comment:
I'm noticing this broken UI, blue block overlapping probably because `solutions` text is longer than all previous uses.

![image](https://user-images.githubusercontent.com/59381432/203394597-0b58b22c-577d-433f-9f66-b7acbd989eb7.png)

Reply:
Ah weird, I just noticed this is intermittent. I'll push up a fix shortly!